Topic3/src/org/maugiam/tdd/main/AmountConverter.java

```diff
@@ -0,0 +1,79 @@
+package org.maugiam.tdd.main;
+
+import java.util.HashMap;
+
+public class AmountConverter {
+
+	protected HashMap<Integer, String> numbers;
+	protected HashMap<Integer, String> tenNames;
+
+	public AmountConverter() {

```

Comment:
Warning: Since it is an utility class, I think it makes more sense to declare this class as static, like you did with the ArabicRomanConverter class.